# target LTM data: episode, entity, stream
string target

# semantic type
# only valid when 'target' in [entity, entity_trail, stream]
string semantic_type

# json query based on MongoDB style
string json

bool logging
---
# matching uids
uint32[] episodes
ltm/QueryResult[] streams
ltm/QueryResult[] entities
ltm/QueryResult[] entities_trail
